src/wallet/wallet.cpp

```diff
@@ -235,7 +235,7 @@ std::shared_ptr<CWallet> LoadWallet(interfaces::Chain& chain, const std::string&
 {
     auto result = WITH_LOCK(g_loading_wallet_mutex, return g_loading_wallet_set.insert(name));
     if (!result.second) {
-        error = Untranslated("Wallet already being loading.");
+        error = Untranslated("Wallet is already currently being loaded.");
```

Comment:
```suggestion
        error = Untranslated("Wallet already loading");
```

Reply:
"being loaded" is more precise since "loading" implies that the wallet is acting instead of being acted on. So probably "Wallet already being loaded" is the best fit.

Reply:
I tend to agree with @jonatack here. Shorter is better here.